x-pack/plugins/apm/public/components/app/TransactionDetails/view.tsx

```diff
@@ -63,7 +64,16 @@ export function TransactionDetailsView({ urlParams, location }: Props) {
 
       <TransactionDetailsRequest
         urlParams={urlParams}
-        render={({ data: transaction }) => {
+        render={({ data: transaction, status }) => {
+          if (isEmpty(transaction)) {
```

Comment:
Doesn't the endpoint either return a transaction of type Transaction, or null?

Reply:
i.e. `if (transaction)` or `if (transaction !== null)` both seem sufficient and no need for `isEmpty`, was what I was wondering with that question.

Reply:
Good point! I have changed accordingly.